rdeps.xml

```diff
@@ -0,0 +1,18 @@
+
+<dependencies>
+  <fileversion>0.1</fileversion>
+  <os names="Windows">
+    <vcpkg>
+      <pkg name="google test">gtest:x64-windows</pkg>
+      <pkg name="OpenBLAS">openblas:x64-windows</pkg>
+    </vcpkg>
+  </os>
+  <os names="all">
+    <pip>
+      <pkg name="PyYAML">pyyaml</pkg>
+      <pkg name="six">six</pkg>
+      <pkg name="wheel">wheel</pkg>
+      <pkg name="virtualenv">virtualenv</pkg>
```

Comment:
Do you need wheel and virtualenv?

Reply:
No, I should be able to get rid of them. Do you happen to know if `six` is used for something other than a virtualenv dependency in rocBLAS?

Reply:
If you grep for six and nothing shows I would try to remove it.